x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/expression/Nullability.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+package org.elasticsearch.xpack.sql.expression;
+
+import org.elasticsearch.xpack.sql.SqlIllegalArgumentException;
+
+public enum Nullability {
+    TRUE((byte) 1),    // Whether the expression can become null
```

Comment:
I'd remove the bitmask - it doesn't seem to add much value (see the previous method suggestions for implementing and); shorter and clearer than using bits.